xgi/drawing/draw.py

```diff
@@ -1658,45 +1681,86 @@ def draw_dihypergraph(
     See Also
     --------
     draw
-    draw_nodes
-    draw_node_labels
+    draw_multilayer
 
     """
-    if not isinstance(DH, DiHypergraph):
-        raise XGIError("The input must be a DiHypergraph")
+    if isinstance(H, DiHypergraph):
+        from ..convert import to_hypergraph
+
+        DH = H.copy()
+        H = to_hypergraph(DH)
+
+    if not isinstance(H, Hypergraph):
+        raise XGIError("The input must be a Hypergraph")
+
+    if not max_order:
+        edge_ids = list(H.edges)
+    else:
+        edge_ids = list(H.edges.filterby("order", max_order, "leq"))
 
     settings = {
-        "min_node_size": 5,
-        "max_node_size": 30,
         "min_node_lw": 0,
         "max_node_lw": 5,
-        "min_lines_lw": 0,
-        "max_lines_lw": 50,
+        "min_node_size": 5,
+        "max_node_size": 30,
+        "min_edge_marker_lw": 0,
+        "max_edge_marker_lw": 5,
+        "min_edge_marker_size": 0,
+        "max_edge_marker_size": 5,
+        "min_dyad_lw": 1,
+        "max_dyad_lw": 5,
+        "node_fc_cmap": cm.Reds,
+        "node_ec_cmap": cm.RdBu,
+        "dyad_color_cmap": cm.Blues,
+        "edge_marker_fc_cmap": cm.Greys,
+        "edge_marker_ec_cmap": cm.Blues,
```

Comment:
already in arguments above, or not used i think

Reply:
Fixed.